testgrid/config.yaml

```diff
@@ -2148,6 +2148,12 @@ test_groups:
 # kubeadm-kinder tests
 - name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-stable-master
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-kinder-upgrade-stable-master
+- name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-1-13-1-14
+  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-kinder-upgrade-1-13-1-14
+- name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-1-12-1-13
+  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-kinder-upgrade-1-12-1-13
+- name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-1.11-1.12
```

Comment:
`test_group_name`s should use dashes:
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/test-infra/12400/pull-test-infra-bazel/1122927985083551745

dashboard `name`s are allowed to use dots too:
https://k8s-testgrid.appspot.com/sig-cluster-lifecycle-all#kubeadm-kinder-upgrade-stable-master